src/main/sarl/fr/utbm/info/ia51/crowdproject/motion/MotionAlgorithms.sarl

```diff
@@ -43,9 +47,23 @@ class KinematicWanderAlgorithm implements WanderAlgorithm {
  */
 class SteeringWanderAlgorithm implements WanderAlgorithm {
 
-	override run(emitter : UUID, currentPosition : Vector2) : MotionQuery {
-		val query = new MotionQuery(emitter, MotionBehavior::STEERING)
-		// TODO
-		return query
+	override run(emitter : UUID, currentPosition : Vector2, orientation : double, velocity : Vector2) : MotionQuery {
```

Comment:
Yay, except passing informations the agent about is a bad idea. A simple recalculation with cos/sin is a better idea. It's a small calculus for a better encapsulation.